ingest/src/main/java/com/microsoft/azure/kusto/ingest/IngestionMapping.java

```diff
@@ -0,0 +1,12 @@
+package com.microsoft.azure.kusto.ingest;
+
+public class IngestionMapping {
+    public INGESTION_MAPPING_KIND IngestionMappingKind;
+    public String IngestionMappingReference;
+
+    public IngestionMapping() {
+        IngestionMappingKind = INGESTION_MAPPING_KIND.unknown;
+    }
+
+    public enum INGESTION_MAPPING_KIND {unknown, csv, json, avro}
```

Comment:
why is this caps? https://google.github.io/styleguide/javaguide.html#s4.8.1-enum-classes ?